### DISCUSSION TITLE
Whole system glamor shot

### DISCUSSION THREAD

**Discussion by jkominek** [jkominek/piano-conversion#30] [Show and tell]
![IMG_2176_cropped](https://user-images.githubusercontent.com/885262/104410823-5cdc6d00-5526-11eb-9265-422704886a5f.jpg)

LED power board on the left with +15V and +48V coming in, an ADC board attached to one sensor and the main board, and then the main board with power coming in on the right, and a Qwiic cable connecting it to an (optional) MIDI board. The MIDI board has some consumer MIDI-USB adapter plugged in. If I'd soldered on another Qwiic header, I'd have hooked up an LCD display for the glamor shot. And I just realized I didn't plug the USB header into the main board for the picture. Oh well.

Multiply the ADC boards by 4-8, and the sensors by 88-176, and you'd have a complete set of hardware. That might even work!

**Comment by davidedelvento** [comment]
I am speechless! Congratulations! Are you posting this on pianoworld? If not, do you mind if I do?

**Reply by jkominek** [reply]
Feel free to post it, you'd be doing me a favor as I'm trying to post on forums and such less during the work week. 😬

**Comment by pianogabe** [comment]
Looks great, congrats so far!
